docs/source/getting_started/installation.rst

```diff
@@ -28,24 +28,19 @@ You can install vLLM using pip:
 
 .. note::
 
-    As of now, vLLM's binaries are compiled on CUDA 12.1 by default.
-    However, you can install vLLM with CUDA 11.8 by running:
+    As of now, vLLM's binaries are compiled with CUDA 12.1 and public PyTorch release versions by default.
+    We also provide vLLM binaries compiled with CUDA 11.8 and public PyTorch release versions:
 
     .. code-block:: console
 
         $ # Install vLLM with CUDA 11.8.
-        $ export VLLM_VERSION=0.2.4
+        $ export VLLM_VERSION=0.4.0
```

Comment:
Is there any way that we can keep this version number updated?

Reply:
emmm that might need something like `rst` file templates. I don't think we need to go that far. Even if the number is outdated, users should know they need to change it according to their need.